platform/darwin/src/MGLOpenGLStyleLayer.mm

```diff
@@ -101,7 +101,7 @@ - (instancetype)initWithIdentifier:(NSString *)identifier {
                                                             MGLPrepareCustomStyleLayer,
                                                             MGLDrawCustomStyleLayer,
                                                             MGLFinishCustomStyleLayer,
-                                                            (__bridge void *)self);
+                                                            (__bridge_retained void *)self);
```

Comment:
It's a corner case, but: this will leak a `MGLOpenGLStyleLayer` that's `init`'d but never added to a map.

Reply:
😞 I knew I was missing something! Moved to : https://github.com/mapbox/mapbox-gl-native/issues/10771